servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/CapturedContextProviders.java

```diff
@@ -26,13 +26,11 @@
 import static java.util.Collections.unmodifiableList;
 
 final class CapturedContextProviders {
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(CapturedContextProviders.class);
     private static final List<CapturedContextProvider> PROVIDERS;
 
     static {
         final ClassLoader classLoader = CapturedContextProviders.class.getClassLoader();
-        PROVIDERS = loadProviders(CapturedContextProvider.class, classLoader, LOGGER);
+        PROVIDERS = loadProviders(CapturedContextProvider.class, classLoader);
```

Comment:
It might be helpful to add a try-catch block inside this and/or `AsyncContext` static block to log unexpected exceptions, because JDK doesn't do a good job of making those exceptions visible, it simply says it could not link/load/find a class. If there is a risk to use a logger in that case, we can use stdout.

Reply:
Good idea.